test/test_nn.py

```diff
@@ -8824,6 +8824,27 @@ def test_ReflectionPad2d_large(self, device):
 
             self.assertEqual(x.grad, ref_x.grad)
 
+    @onlyCUDA   # Test if CPU and GPU results match with deterministic mode on
+    def test_ReflectionPad2d_large_deterministic(self, device):
+        torch.use_deterministic_algorithms(True)
```

Comment:
will this be reset to the previous value after this test is run?

Reply:
Sorry this is probably a stupid question. WDYM? Wil what be reset? I don't program in python all that often so maybe im missing something obvious.

Reply:
I'm concerned that this setting may be unintentionally sticky and applied to other tests if e.g., multiple tests from this file are run

Reply:
> I'm concerned that this setting may be unintentionally sticky and applied to other tests if e.g., multiple tests from this file are run

Oh I see. I think I should be able to just recover it's initial state? I'll push something like that upwards.